EXAMPLES.md

```diff
@@ -562,6 +562,8 @@ service:
   type: LoadBalancer
   annotations:
     # This will tell DigitalOcean to enable the proxy protocol.
+    # Note that only REGIONAL type loadbalancers are supported.
+    # service.beta.kubernetes.io/do-loadbalancer-type: "REGIONAL"
```

Comment:
```suggestion
    service.beta.kubernetes.io/do-loadbalancer-type: "REGIONAL"
```
If this is the only type supported, shouldn't this be uncommented?

Reply:
There are multiple types. There is REGIONAL_NETWORK, REGIONAL and GLOBAL.

* [docs](https://docs.digitalocean.com/reference/api/digitalocean/#tag/Load-Balancers/operation/loadBalancers_create)

I left it commented to guide the user if they set the other 2.